types/index.d.ts

```diff
@@ -13,20 +13,26 @@ export interface ChalkOptions {
 	level?: Level;
 }
 
-export interface Chalk {
+export interface ChalkConstructor extends Chalk {
 	new (options?: ChalkOptions): Chalk;
 	(options?: ChalkOptions): Chalk;
+}
+
+export interface ColorSupport {
+	level: Level;
+	hasBasic: boolean;
+	has256: boolean;
+	has16m: boolean;
+}
+
+export interface Chalk {
 	(...text: string[]): string;
 	(text: TemplateStringsArray, ...placeholders: string[]): string;
-	constructor: Chalk;
+	new (options?: ChalkOptions): Chalk;
+	(options: ChalkOptions): Chalk;
```

Comment:
Why are these added here? They're only valid for `chalk.constructor()`

Reply:
I missed moving them when I pulled ChalkConstructor out 😬